app/assets/stylesheets/mobile/header.scss

```diff
@@ -163,6 +162,16 @@ header {
 }
 
 /* This class is added when the user open the drawer */
-#app.draw > * {
-          transform: translateX(-80%);
+.draw {
+    #drawer, #main_nav {
+        transition: transform 0.3s ease;
+        transform: translateX(-80%);
+    }
+}
+
+#drawer, #main_nav {
+    z-index: 10;
```

Comment:
Line should be indented 2 spaces, but was indented 4 spaces<br>Properties should be ordered transform, transition, z-index